3_ProtectionProfile/PP config.adoc

```diff
@@ -21,15 +21,15 @@ The purpose of a PP-Configuration is to define a Target of Evaluation (TOE) that
 
 This PP-Configuration is identified as follows:
 
-* PP-Configuration for Protection Profile for Mobile Device Fundamentals and collaborative Protection Profile Module for Mobile biometric enrolment and verification - for unlocking the device - version 0.8, January 31, 2019
+* PP-Configuration for Protection Profile for Mobile Device Fundamentals and collaborative PP-Module for Mobile biometric enrolment and verification - for unlocking the device - version 0.8, May 01, 2019
 * As a shorthand reference, it can be identified as "BIO+MDF"
```

Comment:
I would suggest that
a) Add a prefix to identify this as a PP-Configuration. If we can establish such a convention it will assist schemes / CCDB determine the type of document when they add it to their databases
b) Potential add a number at the end, since in theory we may have more than one  PP-Configurations that are similar - So, for example either the module or MDFPP may rev, and so we would need another PP-Configuration for each revision.

So, I suggest "PPC+BIO+MDF+01"

Reply:
I have edited the short name but used PPC+MDF+BIO+01 instead so the base PP would be first and the PP-Module(s) would come after. I thought precedence of base to Module should be shown in the PP-Config name (as it currently is in the title as well).